### ISSUE TITLE
Append the first 20 (or so) characters to SMS sent/delivered.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#64]
```
What steps will reproduce the problem?
1. Send SMS.

What is the expected output? What do you see instead?
It would be nice to see what SMS was sent/delivered. Especially if sending to 
multiple people and there is a delay in getting those out as mentioned in other 
tickets. But then when that finally gets sent and delivered to see which ones. 
So the first 20 chars or so would help identify which ones.

What version of the product are you using? On what operating system?
1.7.3

Please provide any additional information below.
Enhancement.

```

Original issue reported on code.google.com by `i.vita...@gmail.com` on 27 Jan 2011 at 6:29

**Comment by GoogleCodeExporter** [comment]
```
The first 20 chars of the SMS. (Sorry).
```

Original comment by `i.vita...@gmail.com` on 27 Jan 2011 at 6:29

**Comment by GoogleCodeExporter** [comment]
```
We should also include the recipient in the delivery notification.
```

Original comment by `fschm...@gmail.com` on 27 Jan 2011 at 9:02
- Changed state: **Accepted**
- Added labels: **Priority-Low**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
For the sent notification it's not possible because the intent only contents 
the result, not the message.
For the received notification I haven't the body message, but I'll try to 
retrieve a message id...
```

Original comment by `Florent....@gmail.com` on 27 Jan 2011 at 9:09

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 28 Jan 2011 at 7:19

**Comment by GoogleCodeExporter** [comment]
```
Wasn't so easy as I first thought. Android documentation is sometimes really 
bad. ;-)
But now I am happy and satisfied with the result. We can now check if all parts 
of a multipart SMS Message have been sent/delivered. The notifications now show 
for WHICH messages they are meant.

Changes include:
- smsMap HashMap for all SMS that have outstanding notifications.
- extended the Sms Class for use with the map above


```

Original comment by `fschm...@gmail.com` on 29 Jan 2011 at 2:25
- Changed state: **Started**
- Added labels: **Priority-Medium**
- Removed labels: **Priority-Low**

Attachments:
- [issue64.bundle](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-64/comment-5/issue64.bundle)

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 29 Jan 2011 at 4:01
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.8
```

Original comment by `Florent....@gmail.com` on 29 Jan 2011 at 10:57
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
